test/test_files/tinysnb/function/cast.test

```diff
@@ -575,3 +574,12 @@ False
 ---- 1
 170141183460469231731687303715884105728.000000|-4324324
 
+-LOG CastNegInt128ToFloatDouble
```

Comment:
Can you confirm whether we have test cases:
uint->double/float, double/flat->uint

Reply:
we have `LOG CastUnsignedIntToFLoatDouble` and `-LOG CastDoubleToUInt`